srcpkgs/mmv-go/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'mmv-go'
+pkgname=mmv-go
+version=0.1.6
+revision=1
+build_style=go
+go_import_path=github.com/itchyny/mmv/cmd/mmv@latest
+short_desc="Rename and move multiple files at once with your editor"
+maintainer="anelki <akierig@fastmail.de>"
+license="MIT"
+homepage="https://github.com/itchyny/mmv"
+distfiles="https://github.com/itchyny/mmv/archive/refs/tags/v${version}.tar.gz"
+checksum=18d181e6a53437ad18c4409f3c423bc23687152a17b53a605dde13dfffb0d215
+
+post_install() {
+	mv ${DESTDIR}/usr/bin/mmv ${DESTDIR}/usr/bin/mmv-go
```

Comment:
```suggestion
	mv ${DESTDIR}/usr/bin/mmv{,-go}
```
or, if you prefer,
```suggestion
	mv ${DESTDIR}/usr/bin/{mmv,mmv-go}
```

Reply:
I don't think that's necessary or more readable

Reply:
I prefer it for brevity, but it's no big deal.